hathor/indexes/manager.py

```diff
@@ -35,13 +38,21 @@
 logger = get_logger()
 
 
+class _IndexFilter(Enum):
+    ALL = auto()  # block or tx, voided or not
+    VALID_BLOCKS = auto()  # only blocks that are not voided
```

Comment:
What about soft-voided blocks?

Reply:
soft-voided blocks don't exist, soft-voided only apply to transactions

Reply:
I know we don't have soft-voided blocks. I'm wondering whether our current implementation would support them. But there's nothing to do here anyway. Thanks!